integration/kube/fixtures.go

```diff
@@ -72,7 +72,7 @@ func ProxyClient(cfg ProxyConfig) (*kubernetes.Clientset, *rest.Config, error) {
 	if err != nil {
 		return nil, nil, trace.Wrap(err)
 	}
-	caCert, signer, err := authServer.GetKeyStore().GetTLSCertAndSigner(ca)
+	caCert, signer, err := authServer.GetKeyStore().GetTLSCertAndSigner(context.TODO(), ca)
```

Comment:
There's already a context.Background created here on line 68, could you hoist a `context.TODO()` or `context.Background()` to the first line of this function and then use the same context in both places ?